### DISCUSSION TITLE
[0.73/0.74] Android `minSdk` Bump announcement

### DISCUSSION THREAD

**Discussion by cortinico** [react-native-community/discussions-and-proposals#740] [Announcements]
**tl;dr: React Native 0.73 will be the last version to support Android 5.0 (API 21). React Native 0.74 will bump the min SDK version to Android 6.0 (API 23)**

Hi all,

We're sharing this announcement to inform our community that we're about to bump the Min SDK version for React Native.

The upcoming version of React Native, version 0.73, will be the **last** version to support Android 5.0 (API level 21). The following minor version, 0.74, will bump the Min SDK version to 23.

React Native 0.73 will follow our usual [release support window](https://github.com/reactwg/react-native-releases/#which-versions-are-currently-supported), meaning that it will be officially supported till the stable release of React Native 0.76.

We're sharing this announcement now to give **plenty of time** to app developers to properly community this change to your users, if necessary.

## Why we're doing this change

Android 5.0 was released on [Nov 4th 2014](https://en.wikipedia.org/wiki/Android_Lollipop), and will be 10 years old in 2024.

The Play Store statistics from Oct 2023 shows how Android 5.0/5.1 users account for **~1.4%** of the active devices, a number which is going to decrease as time passes:

![Screenshot 2023-11-23 at 11 51 12](https://github.com/react-native-community/discussions-and-proposals/assets/3001957/61794bd2-511d-4916-99df-e08be412a8b2)

Google [already recommended](https://twitter.com/minsdkversion) to bump Min SDK to 23 in 2019. We haven't bumped the Min SDK so far to increase the number of users you can support with React Native apps. By dropping support for older versions of Android, we're able to offer you a better React Native experience and simplify our codebase.

## What does this mean for me?

We believe this change won't affect most developers and is part of the natural mobile update cycles.

In the rare circumstance where you're targeting custom hardware or low-end devices, this change means that users of your applications on Android 5.0/5.1 won't be able to receive **updates** of your apps if you update your project beyond React Native 0.73.

In those cases the recommendation is to use React Native 0.73 as long as possible, or consider other mobile solutions instead.

**Comment by dws-shivansh** [comment]
amazing news
